model/src/main/java/com/ge/predix/acs/model/Policy.java

```diff
@@ -67,6 +69,21 @@ public void setConditions(final List<Condition> conditions) {
         this.conditions = conditions;
     }
 
+    /**
+     * @return the obligationIds
+     */
+    public Set<String> getObligationIds() {
```

Comment:
did we discuss/decide if obligations have a expected order of processing for PDP?

Reply:
we decided not to implement that functionality in order to make the mvp as simple as possible

Reply:
in that case, use a concrete Set implementation which preserves order - LinkedHashSet .

Reply:
LinkedHashSet. implementing with linkedhashset